frontend/src/components/StateTooltip.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import InfoIcon from '@material-ui/icons/Info';
+
+class StateTooltip extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const href = 'https://codalab-worksheets.readthedocs.io/en/latest/features/bundles/states';
+        const states = [
```

Comment:
Let's use this from constants: https://github.com/codalab/codalab-worksheets/blob/ed58812c3ecc784eb42985271f360c5b6a3aa18d/frontend/src/constants.js#L42-L55